server/controllers/teacher/getEnrolledStudents.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Response } from 'express';
+import { getEnrolledStudentsQuery } from '../../database/queries';
+import { UserAuth } from '../../interfaces';
+
+export default async ({ params: { quizId }, user }:UserAuth, res:Response, next:NextFunction) => {
+  const { userId: teacherId } = user;
+  try {
+    const { rows: enrolledStudents } = await getEnrolledStudentsQuery(quizId, teacherId);
+    res.json(enrolledStudents);
```

Comment:
We've agreed on a specific form for data retrieving
```suggestion
    res.json({ data: enrolledStudents, message: 'Enrolled students for quiz retrieved successfully' });
```